numpy/_core/tests/test_einsum.py

```diff
@@ -816,7 +816,7 @@ def test_einsum_fixedstridebug(self):
         tp = np.tensordot(A,  B,  axes=(0,  0))
         assert_equal(es,  tp)
         # The following is the original test case from the bug report,
-        # made repeatable by changing random arrays to aranges.
+        # made repeatable by changing random arrays to arranges.
```

Comment:
revert this

Reply:
good catch, adding `arange` and `aranges` to whitelist